services/app-web/src/hooks/useString.ts

```diff
@@ -0,0 +1,15 @@
+import { useLDClient } from 'launchdarkly-react-client-sdk'
+import { featureFlags } from '../common-code/featureFlags'
+
+export const useString = () => {
```

Comment:
Do we plan on using this string long-term? Or will it be removed once this flag is gone?

If we are keeping it for the long run, I would like this hook to be named something like `useStringConstants`.
```suggestion
export const useStringConstants = () => {
```

Reply:
I'd like to keep it around, at least until we have a real solution for strings in the app, so I'll rename it.